swarmion.code-workspace

```diff
@@ -28,7 +28,7 @@
     "[markdown][mdx]": {
       "editor.wordWrap": "on"
     },
-    "typescript.tsdk": "node_modules/typescript/lib",
+    "typescript.tsdk": "swarmion root/node_modules/typescript/lib",
```

Comment:
We should put the following instead so that if we rename the workspace it still works:
```suggestion
    "typescript.tsdk": "./node_modules/typescript/lib",
```

Reply:
I tried this and it did not work @adriencaccia :/ Instead it fell back on the default vscode TS version, which is 4.8.4 and made my debugging a lot harder

Reply:
Alright let's keep that!